cumulusci/tasks/salesforce/insert_record.py

```diff
@@ -13,18 +13,27 @@ class InsertRecord(BaseSalesforceApiTask):
     task_options = {
         "object": {"description": "An sObject type to insert", "required": True},
         "values": {
-            "description": "Field names and values in the format 'aa:bb,cc:dd'",
+            "description": "Field names and values in the format 'aa:bb,cc:dd', or a YAML dict",
             "required": True,
         },
+        "tooling": {"description": "If True, use the Tooling API instead of REST API."},
     }
 
     def _init_options(self, kwargs):
         super()._init_options(kwargs)
-        self.values = process_list_of_pairs_dict_arg(self.options["values"])
+        if type(self.options["values"]) is str:
```

Comment:
process_list_of_pairs_dict_arg already handles values of type dict and str...is there something else you're trying to handle here?

Reply:
There is not. I just didn't know that. I'll update the task doc instead of making this change.